.github/workflows/doc-build.yml

```diff
@@ -109,111 +109,9 @@ jobs:
         lcov --remove doc-coverage.info \*/deprecated > new.info
         genhtml --no-function-coverage --no-branch-coverage new.info -o coverage-report
 
-    - name: compress-docs
-      run: |
-        tar cfJ html-output.tar.xz --directory=doc/_build html
-        tar cfJ api-output.tar.xz --directory=doc/_build html/doxygen/html
-        tar cfJ api-coverage.tar.xz coverage-report
-
-    - name: upload-build
-      uses: actions/upload-artifact@v3
-      with:
-        name: html-output
-        path: html-output.tar.xz
-
     - name: upload-api-coverage
       uses: actions/upload-artifact@v3
       with:
         name: api-coverage
-        path: api-coverage.tar.xz
```

Comment:
you are aware this is not about code coverage, this just for documentation, right?

Reply:
I wanted to collect information about uncovered functions from documentation perspective.
I've just only verify content of new.info, because html report is not usable from python script perspective.